### PR TITLE
(setup-redesign): add stepper to setup

### LINKED ISSUE #711: Setup Redesign: add Header / Stepper to Setup
Add to `SetupContainer`

grafik

### PR DESCRIPTION
**Pull request by cstenglein**
fixes #711

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (escapedcat) [inline .idea/modules.xml:1]: `.idea` files here on purpose?

File: src/pages/Setup/Stepper.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <div className="w-full">
+      <ul className="lg:flex hidden justify-center items-center gap-4">
+        {Array.from({ length: NUMBER_OF_STEPS }).map((_, index: number) => (
+          <>
+            <li
+              className={`list-none text-center flex flex-col ${index < currentStep ? "opacity-100" : "opacity-30"}`}
+              key={index}
+            >
+              <span className="font-bold">Step {index + 1}</span>
```
Comment (escapedcat) [inline src/pages/Setup/Stepper.tsx:21]: Should "Step" be translated?

File: src/pages/Setup/Stepper.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+              <span className="text-sm">
+                {t(`setup.step.step_${index + 1}_description`)}
+              </span>
+            </li>
+            {index < NUMBER_OF_STEPS - 1 && (
```
Comment (escapedcat) [inline src/pages/Setup/Stepper.tsx:26]: Not sure about this. Looks not semantical.  Should be to only be `li`s within `ul` I believe.
Could the line be achieved with i.e. [`:after`](https://stackoverflow.com/a/75512805/1667461) on the `ul`?
Like make it a full with block element with a border or background, position it absolute maybe and cover it with the li's?
Do you know what I mean?

Comment (cstenglein) [reply src/pages/Setup/Stepper.tsx:26]: Tried to do it with `after` and stuff like `:not(:last-child)` to not target the last `li`.
Unfortunately didn't work out as expected, so I left it at that.

Comment (cstenglein) [reply src/pages/Setup/Stepper.tsx:26]: Styling the `ul` seemed like a good idea at first, but then you would need to style all the `li` and the line would be too close to the elements itself.

Comment (escapedcat) [reply src/pages/Setup/Stepper.tsx:28]: Could give it a try as well. There must be nicer way :)

Comment (cstenglein) [reply src/pages/Setup/Stepper.tsx:28]: Should we merge this one then? In the end, it's looking good & the 3 extra elements do not put a burden on performance or anything.

**Pull request merged**